app/helpers/application_helper/button/auth_key_pair_cloud_delete.rb

```diff
@@ -0,0 +1,9 @@
+class ApplicationHelper::Button::AuthKeyPairCloudDelete < ApplicationHelper::Button::Basic
+  needs :@record
+
+  def disabled?
+    return true if @record.validate_delete_key_pair[:available]
+    @error_message = _('Deletion is unavailable for this keypair.')
+    @error_message.present?
+  end
```

Comment:
So the `AuthKeyPairCloudDelete` button is meant to be always disabled? Am I reading the logic correctly here?